tutorials/Application.startManyManifests.md

```diff
@@ -0,0 +1,29 @@
+Asynchronously starts a batch of applications given an array of application identifiers and manifestUrls.
+Returns once the RVM receives the message, not when the applications are running.
+# Example
+```js
+
+const applicationInfoArray = [
+    {
+        "uuid": `App-1`,
+        "manifestUrl": `http://localhost:5555/app1.json`,
```

Comment:
:neckbeard: this is not copy paste runnable, but we since we lack a good public app....